src/runtime/components/CldVideoPlayer.vue

```diff
@@ -67,6 +68,7 @@ export type CldVideoPlayerProps = Pick<
   videoRef?: { value: HTMLVideoElement | null };
   quality?: string | number;
   width: string | number;
+  config: Record<string, any>;
```

Comment:
looks like we're missing an optional flag here: `config?:`

Reply:
Good catch!